src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/Node/JsonArrayConverter.cs

```diff
@@ -20,8 +20,6 @@ public override void Write(Utf8JsonWriter writer, JsonArray value, JsonSerialize
             {
                 case JsonTokenType.StartArray:
                     return ReadList(ref reader, options.GetNodeOptions());
-                case JsonTokenType.Null:
-                    return null;
```

Comment:
Is this because the serializer handles `null`?

Reply:
Yes. At one point in the prototyping this was necessary; test code coverage confirmed it wasn't necessary.